aiohttp/web_fileresponse.py

```diff
@@ -266,7 +266,7 @@ async def prepare(self, request: "BaseRequest") -> Optional[AbstractStreamWriter
             )
 
         # If we are sending 0 bytes calling sendfile() will throw a ValueError
-        if count == 0 or request.method == hdrs.METH_HEAD or self.status in [204, 304]:
+        if count == 0 or must_be_empty_body(request.method, self.status):
```

Comment:
If the problem here is sending 0 bytes, then that'd include both 304 and HEAD, right?

Reply:
Ignore that one, mixing up the methods.